casanovo/casanovo.py

```diff
@@ -130,13 +138,26 @@ def main(
 
     # Log the active configuration.
     logger.info("Casanovo version %s", str(__version__))
+    logger.debug("mode = %s", mode)
+    logger.debug("model = %s", model)
+    logger.debug("peak_dir = %s", peak_dir)
+    logger.debug("peak_dir_val = %s", peak_dir_val)
+    logger.debug("config = %s", config_fn)
+    logger.debug("output = %s", output)
```

Comment:
**praise:** This is a nice addition that will help us with troubleshooting folks' issues!